ui/src/wizard/ResourceGroupCreateStep.ts

```diff
@@ -21,7 +21,8 @@ export class ResourceGroupCreateStep<T extends types.IResourceGroupWizardContext
         const newName: string = wizardContext.newResourceGroupName!;
         // tslint:disable-next-line:no-non-null-assertion
         const newLocation: string = wizardContext.location!.name!;
-        const resourceClient: ResourceManagementClient = createAzureClient(wizardContext, ResourceManagementClient);
+        const armResources: typeof import('@azure/arm-resources') = await import('@azure/arm-resources');
```

Comment:
This looks pretty tedious to copy/paste if we need to create a client again. How about you add a separate "clients" file of some kind with functions like "createResourceClient" and "createStorageClient". They'll be short functions, but that way we could use autocomplete instead of having to copy/paste these lines when adding new client code

Also, I could see myself copying the "clients" file over to other repos when implementing this